### ISSUE TITLE
Oni crashes on Ubuntu when opening from terminal

### ISSUE THREAD

**Issue by Mukundan314** [onivim/oni#1234]
when I start oni from the terminal it crashes

![terminal_001](https://user-images.githubusercontent.com/30190448/34606786-b144c29e-f236-11e7-9fd8-72dc40e8784c.png)

![oni_002](https://user-images.githubusercontent.com/30190448/34606810-ce2f8920-f236-11e7-8f0a-943c8f261352.png)

platform: Ubuntu 16.04
oni version: v0.2.19
neovim version: v0.2.2

**Comment by bryphe** [comment]
Hi @Mukundan314 ,

Can you please try running with `oni --debug` and share the output of the console log? That should help us narrow down the issue.

Thank you!

**Comment by Mukundan314** [comment]
i am not able to read the console since oni crashes

**Comment by Mukundan314** [comment]
It does crash. I am not able to type or click on anything in the oni window.

**Comment by novmar** [comment]
try to swap into console, and type "fg" (i had same issue, and oni window was "invissible" - so i saw console, but i was unable to click it - because invissible oni windows was over it. i had to switch to console - alt+tab - and type fg. )

**Comment by bryphe** [comment]
Sorry you're both hitting this issue on startup, @Mukundan314 & @novmar !

Usually there are one of two causes:
 - Oni cannot find your `nvim` executable
- The `init.vim` has an error (in which case executing a key press would help).

Some troubleshooting that would be helpful:
- The output of `which nvim` in the terminal
- The output of `nvim --version` in the terminal
- In Oni's debug window, the output of `process.env`

In addition, which install package did you use? The `tar.gz`?

**Comment by novmar** [comment]
Hi
my nvim is fine, - nvim --version 

https://pastebin.com/3Suw3AP6

when i run oni wit --debug, i can see in console boot of oni .. and it freezes in time (08:59:10.814) and then nothing happens. in console, i can see "  [1]+  Stopped                 oni --debug " . Now, oni is totaly unresponsible, and it is also invissible (process is stopped, and it doesn'r redraw window)
if i type "fg" in console, oni continue to boot  (in this try, it is time 08:59:14,941).


my oni is installed by arch yum package https://aur.archlinux.org/packages/oni/ 
and installation is from tar.gz

**Comment by Mukundan314** [comment]
I have oni installed with the deb package and also running `exit` then `fg` in the terminal seem to un freeze the window

my nvim is fine and i currently removed my init.vim

**Comment by JSE018** [comment]
I have the same issue. nvim installation is fine, version 2.2. Installed oni using deb file on ubuntu 16.04. It stops when running from terminal, but is fine otherwise. Please fix it, because it is literally unworkable in linux environment.

**Comment by hoschi** [comment]
Saw the big chat between @bryphe and @someguynamedmatt on gitter. I wanted to develop with Oni already but stuck at a new syntax theme for the last weekends. I never saw problems like these or from @someguynamedmatt and I'm using Oni now for a long time with different nvim versions. Regarding to `npm start` doesn't working this works for me fine, changing `backgroundColor` in `browser/src/UI/components/Tabs.tsx` reloads Oni and applies changes. So here is my environment, hopes this helps to find the differences:

current commit: fef2e10d1abf55bcb4126fbb5ed86cf5dde0ac4d
```
oni-master > nvim -version
NVIM v0.2.2-46-gf185c739b
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wconversion -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -O2 -g -DMIN_LOG_LEVEL=3 -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/tmp/yaourt-tmp-hoschi/aur-neovim-git/src/neovim-git/build/config -I/tmp/yaourt-tmp-hoschi/aur-neovim-git/src/neovim-git/src -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/tmp/yaourt-tmp-hoschi/aur-neovim-git/src/neovim-git/build/src/nvim/auto -I/tmp/yaourt-tmp-hoschi/aur-neovim-git/src/neovim-git/build/include
Übersetzt von hoschi@maxxgubbl

Features: +acl +iconv +jemalloc +tui 
See ":help feature-compile"

          System-vimrc-Datei: "$VIM/sysinit.vim"
     Voreinstellung für $VIM: "/usr/share/nvim"

Run :checkhealth for more info
oni-master > node --version
v8.9.1
oni-master > npm --version
5.5.1
oni-master > yarn --version
1.3.2
oni-master > tsc --version
zsh: correct 'tsc' to 'tc' [nyae]? n
Version 2.6.1
oni-master > which tsc
./node_modules/.bin/tsc
oni-master > cat $PATH
cat: '/home/hoschi/.nvm/versions/node/v8.9.1/bin:/home/hoschi/bin:/home/hoschi/.gem/ruby/2.4.0/bin:/home/hoschi/src/adk-current/sdk/tools:/home/hoschi/src/adk-current/sdk/platform-tools:/usr/share/maven2/bin:/opt/java/bin:/opt/java/db/bin:/opt/java/jre/bin:/home/hoschi/.local/bin:./node_modules/.bin:/home/hoschi/.sdkman/candidates/grails/current/bin:/home/hoschi/bin:/home/hoschi/.gem/ruby/2.4.0/bin:/home/hoschi/src/adk-current/sdk/tools:/home/hoschi/src/adk-current/sdk/platform-tools:/usr/share/maven2/bin:/opt/java/bin:/opt/java/db/bin:/opt/java/jre/bin:/home/hoschi/.local/bin:./node_modules/.bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/opt/bin:/home/hoschi/bin:/opt/bin:/home/hoschi/bin:/opt/bin:/home/hoschi/.fzf/bin:/opt/bin:/home/hoschi/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/opt/bin:/home/hoschi/bin:/opt/bin': Datei oder Verzeichnis nicht gefunden
oni-master > pacman -Q | agi nvim
zsh: correct 'nvim' to 'vim' [nyae]? n
oni-master > pacman -Q | agi vim 
gvim 8.0.1415-1
neovim-git 0.2.1.r46.gf185c739b-1
python-neovim-git 0.2.0.r1.g936110a-1
python2-neovim-git 0.2.0.r1.g936110a-1
vim-runtime 8.0.1415-1

```

**Comment by hoschi** [comment]
`:CheckHealth` output run within Oni:

```

health#nvim#check
========================================================================
## Configuration
  - OK: no issues found

## Performance
  - OK: Build type: RelWithDebInfo

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=^H
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $XTERM_VERSION=''
  - INFO: $TERM_PROGRAM=''
  - INFO: $SSH_TTY=''

health#provider#check
========================================================================
## Clipboard (optional)
  - OK: Clipboard tool found: xsel

## Python 2 provider (optional)
  - INFO: Using: g:python_host_prog = "/usr/bin/python2"
  - INFO: Executable: /usr/bin/python2
  - INFO: Python2 version: 2.7.14
  - INFO: python2-neovim version: 0.2.1dev
  - OK: Latest python2-neovim is installed: 0.2.0

## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Multiple python3 executables found.  Set `g:python3_host_prog` to avoid surprises.
  - INFO: Executable: /usr/bin/python3
  - INFO: Other python executable: /bin/python3
  - INFO: Python3 version: 3.6.4
  - INFO: python3-neovim version: 0.2.1dev
  - OK: Latest python3-neovim is installed: 0.2.0

## Ruby provider (optional)
  - INFO: Ruby: ruby 2.4.3p205 (2017-12-14 revision 61247) [x86_64-linux]
  - WARNING: Missing "neovim" gem.
    - ADVICE:
      - Run in shell: gem install neovim
      - Is the gem bin directory in $PATH? Check `gem environment`.
      - If you are using rvm/rbenv/chruby, try "rehashing".

## Node provider (optional)
  - INFO: Node: v8.9.1
  - INFO: Host: /home/hoschi/.nvm/versions/node/v8.9.1/bin/neovim-node-host
  - WARNING: Package "neovim" is out-of-date. Installed: 3.5.0, latest: 3.5.2
    - ADVICE:
      - Run in shell: npm update neovim
```

**Comment by bryphe** [comment]
Thanks all for the details! I'm able to reproduce this on my Arch Linux box in the built repo, by running `node_modules /.bin/electron lib/main/src/main.js`. Not sure what the issue is yet.

**Comment by bryphe** [comment]
Resolved with #1368 - planning on releasing a new build on Monday w/ the fix.
